### PR TITLE
Fix worker build

### PR DESCRIPTION
**Pull request by hannessolo**

### CODE REVIEW

File: renderer/_default.page.client.jsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const { hydrationPromise } = useClientRouter({
     
     const customParamsString = sessionStorage.getItem('customParams');
     if (!customParamsString) {
-      if (import.meta.env.PROD) {
+      if (import.meta.env.MODE !== 'development') {
```
Comment (hannessolo) [inline renderer/_default.page.client.jsx:13]: Not sure if this is right. I got a merge conflict and went with my version.

File: renderer/_default.page.server.jsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ async function render(pageContext) {
     </PageLayout>
   </StyleSheetManager>
   
-  if (import.meta.env.PROD) {
+  if (import.meta.env.MODE === 'worker') {
```
Comment (hannessolo) [inline renderer/_default.page.server.jsx:25]: Not sure if this is right. I got a merge conflict and went with my version.

**Review by icaraps — CHANGES_REQUESTED** [review]
Can you apply https://github.com/adobe-rnd/franklin-esr/pull/29/commits/2f6c33198ffa5b29f1d6aee09e1ca446d05988ef ? 

Basically we dont need to specify the worker mode, we define production as being the worker build and else its dev mode. 
I did this change when I noticed that setting the mode to worker was bundling js files in none production mode (leading to doubling the size of the js files because it included eg react development code etc)

**Comment by hannessolo** [comment]
> Can you apply [2f6c331](https://github.com/adobe-rnd/franklin-esr/commit/2f6c33198ffa5b29f1d6aee09e1ca446d05988ef) ?
> 
> Basically we dont need to specify the worker mode, we define production as being the worker build and else its dev mode. I did this change when I noticed that setting the mode to worker was bundling js files in none production mode (leading to doubling the size of the js files because it included eg react development code etc)

Thanks, that makes sense

**Review by icaraps — APPROVED** [review]

**Pull request merged**